src/test/scala/com/goyeau/kubernetes/client/api/ContextProvider.scala

```diff
@@ -0,0 +1,13 @@
+package com.goyeau.kubernetes.client.api
+
+import cats.effect.{Blocker, ContextShift, IO, Timer}
+
+import scala.concurrent.ExecutionContext
+
+trait ContextProvider {
+
+  implicit lazy val ioTimer: Timer[IO]   = IO.timer(ExecutionContext.global)
+  implicit lazy val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
Let's name them `timer` and `contextShift`

Reply:
Done. I have also removed redundant `ContextShift` and `Timer` from *Test clases.